Specs/Core/RuntimeErrorSpec.js

```diff
@@ -19,13 +19,22 @@ describe('Core/RuntimeError', function() {
     });
 
     it('has a stack property', function() {
-        expect(e.stack).toContain('RuntimeErrorSpec.js');
+        if (window.specsUsingRelease) {
+            expect(e.stack).toContain('Specs.js');
+        } else {
+            expect(e.stack).toContain('RuntimeErrorSpec.js');
+        }
     });
 
     it('has a working toString', function() {
         var str = new RuntimeError(testMessage).toString();
 
         expect(str).toContain(name + ': ' + testMessage);
-        expect(str).toContain('Core/RuntimeErrorSpec.js');
+
+        if (window.specsUsingRelease) {
```

Comment:
From the perspective of me writing tests, do I need to be aware of `specsUsingRelease` ? Is it worth documenting before merging?

Reply:
This isn't a new flag, it's been there forever and its a very rare occurrence anyone would have to worry about it.  These particular tests are wonky edge cases. The idea is to not be aware of a difference so that we run the exact same specs in dev as release.